### PR TITLE
[NATALIYA_MYKYTYUK] Tasks 61-70

### PR DESCRIPTION
**Pull request by dnistrova**

### CODE REVIEW

File: tasks/61_spinal-case/spinal-case.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export function spinalCase(str) {
+  const regex = /\s+|_+/g;
+  str = str.replace(/([a-z])([A-Z])/g, '$1 $2');
```
Comment (AlexLarusso) [inline tasks/61_spinal-case/spinal-case.js]: empty line before

File: tasks/61_spinal-case/spinal-case.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  return str.replace(regex, '-').toLowerCase();
```
Comment (AlexLarusso) [inline tasks/61_spinal-case/spinal-case.js]: empty line before

File: tasks/62_swap-cases/swap-cases.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export function swapCases(str) {
+  return str.replace(/([a-z]+)|([A-Z]+)/g, function(match, chr) {
```
Comment (AlexLarusso) [inline tasks/62_swap-cases/swap-cases.js]: You should handle cases when str is not a string.
Use arrow function;

File: tasks/63_insert-string/insert-string.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export function insert(mainStr, str, position) {
+  if (typeof(position) == 'undefined') {
+    position = 0;
+  }
+  if (typeof(str) == 'undefined') {
+    str = '';
+  }
```
Comment (AlexLarusso) [inline tasks/63_insert-string/insert-string.js]: You can use [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) instead.

File: tasks/63_insert-string/insert-string.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+  return mainStr.slice(0, position) + str + mainStr.slice(position);
```
Comment (AlexLarusso) [inline tasks/63_insert-string/insert-string.js]: Use template string instead

File: tasks/63_insert-string/insert-string.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+}
```
Comment (AlexLarusso) [inline tasks/63_insert-string/insert-string.js]: **Changes requested**

File: tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+export function sumFibs(num) {
+  let previous = 0;
+  let current = 1;
+  let result = 0;
+  while (current <= num) {
+    if (current % 2 !== 0) {
+      result += current;
+    }
+    let added = current + previous;
```
Comment (AlexLarusso) [inline tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js]: Move variable declaration to function beginning.

File: tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
+export function sumFibs(num) {
+  let previous = 0;
+  let current = 1;
+  let result = 0;
+  while (current <= num) {
```
Comment (AlexLarusso) [inline tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js]: empty line before

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+export function smallestCommons(arr) {
+  arr.sort(function(a, b) {
```
Comment (AlexLarusso) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: use arrow function

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+    return b - a;
+  });
+
+  const newArr = [];
```
Comment (AlexLarusso) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: Put constant declaration to line 2

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+  for (let i = arr[0]; i >= arr[1]; i--) {
+    newArr.push(i);
+  }
+
+  let quotient = 0;
+  let loop = 1;
+  let n;
```
Comment (AlexLarusso) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: Same. Move to beginning 11-13 lines

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+
+  do {
+    quotient = newArr[0] * loop * newArr[1];
+    for (n = 2; n < newArr.length; n++) {
```
Comment (AlexLarusso) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: move `newArr.length` to const

File: tasks/66_sum-primes/sum-primes.js [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+        primes.push(i);
+        for (let j = i << 1; j <= max; j += i) {
+          arr[j] = true;
+        }
+      }
+    }
+    return primes;
+  }
+
+  const primes = getPrimes(num);
```
Comment (AlexLarusso) [inline tasks/66_sum-primes/sum-primes.js]: Move variable declaration to function beginning.

File: tasks/66_sum-primes/sum-primes.js [hunk 14]
```diff
@@ -0,0 +1,24 @@
+export function sumPrimes(num) {
+  let result = 0;
+
+  function getPrimes(max) {
+    const arr = [];
+    const primes = [];
+    for (let i = 2; i <= max; ++i) {
```
Comment (AlexLarusso) [inline tasks/66_sum-primes/sum-primes.js]: empty line above

File: tasks/66_sum-primes/sum-primes.js [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+      if (!arr[i]) {
+        primes.push(i);
+        for (let j = i << 1; j <= max; j += i) {
```
Comment (AlexLarusso) [inline tasks/66_sum-primes/sum-primes.js]: empty line above

File: tasks/66_sum-primes/sum-primes.js [hunk 16]
```diff
@@ -0,0 +1,24 @@
...
+          arr[j] = true;
+        }
+      }
+    }
+    return primes;
+  }
+
+  const primes = getPrimes(num);
+  for (let p = 0; p < primes.length; p++) {
```
Comment (AlexLarusso) [inline tasks/66_sum-primes/sum-primes.js]: primes.length to const

File: tasks/67_find-elements/find-elements.js [hunk 17]
```diff
@@ -0,0 +1,9 @@
+export function findElement(arr, func) {
+  let num;
+  for (let i = 0; i < arr.length; i++) {
+    if (func(arr[i]) === true) {
```
Comment (AlexLarusso) [inline tasks/67_find-elements/find-elements.js]: Use double exclamation mark instead

File: tasks/67_find-elements/find-elements.js [hunk 18]
```diff
@@ -0,0 +1,9 @@
...
+      num = arr[i];
+      return num;
```
Comment (AlexLarusso) [inline tasks/67_find-elements/find-elements.js]: return arr[i] instead

File: tasks/68_drop-elements/drop-elements.js [hunk 19]
```diff
@@ -0,0 +1,12 @@
+export function dropElements(arr, func) {
+  const times = arr.length;
+  for (let i = 0; i < times; i++) {
```
Comment (AlexLarusso) [inline tasks/68_drop-elements/drop-elements.js]: empty line before

File: tasks/69_unpack-elements/unpack-elements.js [hunk 20]
```diff
@@ -0,0 +1,17 @@
+export function unpackArray(arr) {
+  const flattenedArray = [];
+
+  const flatten = function(arg) {
```
Comment (AlexLarusso) [inline tasks/69_unpack-elements/unpack-elements.js]: Use arrow function, Lift one line up

File: tasks/70_binary-decode/binary-decode.js [hunk 21]
```diff
@@ -0,0 +1,11 @@
+export function binaryDecode(str) {
+  if (!str) {
+    str = '';
+  }
```
Comment (AlexLarusso) [inline tasks/70_binary-decode/binary-decode.js]: Use default parameters

File: tasks/70_binary-decode/binary-decode.js [hunk 22]
```diff
@@ -0,0 +1,11 @@
...
+  let binString = '';
+
+  str.split(' ').map(function(bin) {
```
Comment (AlexLarusso) [inline tasks/70_binary-decode/binary-decode.js]: Use arrow function

File: tasks/70_binary-decode/binary-decode.js [hunk 23]
```diff
@@ -0,0 +1,11 @@
...
+    binString += String.fromCharCode(parseInt(bin, 2));
+  });
+  return binString;
```
Comment (AlexLarusso) [inline tasks/70_binary-decode/binary-decode.js]: Empty line before

**Comment by tarasbobak** [comment]
@AlexLarusso Please merge if you request of changes was fulfilled.

**Pull request merged**